framework/filters/AccessRule.php

```diff
@@ -174,7 +174,7 @@ protected function matchIP($ip)
             return true;
         }
         foreach ($this->ips as $rule) {
-            if ($rule === '*' || $rule === $ip || (($pos = strpos($rule, '*')) !== false && !strncmp($ip, $rule, $pos))) {
+            if ($rule === '*' || $rule === $ip || (($pos = strpos($rule, '*')) !== false && $ip !== null && !strncmp($ip, $rule, $pos))) {
```

Comment:
I know, that condition should be refactored. I'll make an update to a PR if it's going to be adopted.